pkg/graveler/graveler.proto

```diff
@@ -15,6 +15,8 @@ message RepositoryData {
 message BranchData {
   string id = 1;
   string commit_id = 2;
+  string staging_token = 3;
+  repeated string sealed_tokens = 4;
```

Comment:
I think the order of the `sealed_tokens` is worth a documentation. Is the first/last token the newest one?

Reply:
I was just thinking about it - I think that for the commit logic, the best way will be to have it sorted by oldest first. This way, for any new change we can simply override the current changes